lib/fluent/plugin/in_prometheus.rb

```diff
@@ -115,49 +129,40 @@ def shutdown
       super
     end
 
-    class MonitorServlet < WEBrick::HTTPServlet::AbstractServlet
-      def initialize(server, prometheus)
-        @prometheus = prometheus
-      end
+    def all_metrics
+      [200, { 'Content-Type' => ::Prometheus::Client::Formats::Text::CONTENT_TYPE }, ::Prometheus::Client::Formats::Text.marshal(@registry)]
```

Comment:
If we don't use other headers, only return content_type is memory friendly.

Reply:
Right. But it's a temporary implementation. I'll replace it with HTTP helper later(HTTP helper requires a header as this form). So I'll leave it as is.